_posts/2021-04-15-jesse-frost.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Jesse Posner on Flexible Round-Optimized Schnorr Threshold Signatures (FROST)"
```

Comment:
Maybe switch the title and description:

title: "Jesse Posner on FROST"
description: "Jesse Posner explains the Flexible Round-Optimized Schnorr Threshold Signatures (FROST) signature scheme."

So the title is pithy and the description adds more detail.